contracts/auction.sol

```diff
@@ -173,6 +181,16 @@ contract DutchAuction {
         price_exponent = _price_exponent;
     }
 
+    /// @notice Adds account addresses to whitelist.
+    /// @dev Adds account addresses to whitelist.
+    /// @param _bidder_addresses Array of addresses.
+    function addToWhitelist(address[] _bidder_addresses) public isOwner {
+        for (uint32 i = 0; i < _bidder_addresses.length; i++) {
+            assert(_bidder_addresses[i] != 0x0);
```

Comment:
Why do we need this? Couldn't the whitelist in theory be also populated before auction starts?

Reply:
I don't understand - why do we need the `assert`?
The whitelist can be populated befor auction starts - I don't have a stage requirement.

Reply:
I believe we should be able to start populating the whitelist before auction starts.

Reply:
Removed that assert anyway, as per your proposal. Worst case scenario, we whitelist `0x0`